src/mscorlib/shared/System/IO/FileStream.Win32.cs

```diff
@@ -3,14 +3,29 @@
 // See the LICENSE file in the project root for more information.
 
 using Microsoft.Win32.SafeHandles;
-using System.Runtime.InteropServices;
 
 namespace System.IO
 {
     public partial class FileStream : Stream
     {
+        private static bool IsFromAppAvailable()
```

Comment:
I do not think this should be based on whether the library happens to be laying around. If you should be unconditionally true in AppX mode and unconditionally false in regular mode.